web-components/button/button.svelte

```diff
@@ -85,51 +97,69 @@
   .leoButton.isPrimary {
     --bg: var(--color-interaction-button-primary-background);
     --bg-hover: var(--color-light-primary-60);
-    --bg-active: var(--color-light-primary-60);
+    --bg-active: var(--bg-hover);
     --bg-focus: var(--bg);
     --bg-loading: var(--color-light-primary-70);
     --bg-disabled: var(--color-gray-30);
     --color: white;
+
     @theme (dark) {
       --bg-hover: var(--color-dark-primary-20);
       --bg-active: var(--color-dark-primary-20);
       --bg-loading: var(--color-dark-primary-60);
     }
   }
-
   .leoButton.isSecondary {
-    // TODO(petemill): Is transparent right or should it be explicitly white (and black)?
     --bg: transparent;
     --bg-active: --color-gray-20;
     --color: var(--color-text-primary);
-    --color-hover: var(--color-light-primary-70);
+    --color-hover: var(--color-text-interactive);
+    --color-focus: var(--color-text-primary);
     --color-loading: var(--color-gray-70);
     --border-width: 1px;
-    --border-color: var(--color-gray-30);
+    --border-color: var(--color-divider-strong);
     --border-color-focus: transparent;
-    &[disabled] {
+
+    &:disabled {
       opacity: 0.5;
     }
+
     @theme (dark) {
-      --bg: transparent;
-      --color-hover: var(--color-dark-primary-80);
+      --border-color-hover: var(--color-dark-primary-40);
     }
   }
-  .isTertiary {
+  .leoButton.isTertiary {
```

Comment:
Do we need this `.leoButton` selector at all?

Reply:
Yep. because when we extract the CSS, we want `isTertiary` properly scoped to `.leoButton`. It's also consistent with the other `kinds`.